.github/workflows/assign-author.yml

```
@@ -0,0 +1,15 @@
+name: Auto Author Assign
+
+on:
+  pull_request_target:
+    types: [opened, reopened]
```

since we have unassigned issues too

so may want to help filtering those

```suggestion
  issues:
    types: [opened, reopened]
  pull_request_target:
    types: [opened, reopened]
```

https://github.com/toshimaru/auto-author-assign?tab=readme-ov-file#enable-auto-author-assign-for-issues

---

eh, I don't want that, personally. just because I open a bug doesn't mean I'm gonna do it. but a PR is a good indicator! 😄